cmd/helm/install.go

```diff
@@ -193,6 +195,8 @@ func newInstallCmd(c helm.Interface, out io.Writer) *cobra.Command {
 	f.StringVar(&inst.keyFile, "key-file", "", "identify HTTPS client using this SSL key file")
 	f.StringVar(&inst.caFile, "ca-file", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
 	f.BoolVar(&inst.devel, "devel", false, "use development versions, too. Equivalent to version '>0.0.0-a'. If --version is set, this is ignored.")
+	f.Var(&inst.secretFiles, "secrets", "specify secrets in a yaml file (can specify multiple)")
```

Comment:
is there any way we can make it explicit that these flags only work with tiller set to --storage=secret?